.github/ISSUE_TEMPLATE/first_rc_checklist.md

```diff
@@ -0,0 +1,33 @@
+---
+name: First Release Candidate Checklist (maintainer only)
+about: Checklist template for the first release of every series
+title: llvmlite X.Y.Zrc1 Checklist (FIXME)
+labels: task
+
+---
+
+
+## llvmlite X.Y.Z
+
+* [ ] Merge to `main`.
+    - [ ] "remaining Pull-Requests from milestone".
+* [ ] Merge change log changes.
+    - [ ] "PR with changelog entries".
+* [ ] Create X.Y release branch `releaseX.Y`
+* [ ] Annotated tag `vX.Y.Z` on release branch (`llvmlite` tags DO have a `v` prefix).
+* [ ] Build and upload conda packages on buildfarm (check "upload").
+* [ ] Verify packages uploaded to Anaconda Cloud and move to `numba/label/main`.
+* [ ] Build wheels and sdist on the buildfarm.
```

Comment:
```suggestion
* [ ] Build and upload conda packages on buildfarm (check "upload").
* [ ] Build wheels and sdist on the buildfarm (check "upload").
* [ ] Verify packages uploaded to Anaconda Cloud and move to `numba/label/main`.
```
to match Numba's equivalent. For some reason the `sub_rc_checklist.md` in Numba (and llvmlite, below, in this patch) has these the other way around. I don't think it matters way it is, but probably makes sense for them all to be consistent?

Also note the addition of "`(check "upload")` to the `Build wheels and sdist on the buildfarm.` item.

Reply:
Sounds good, better to be consistent.